test/parameter_test.dart

```diff
@@ -85,13 +85,12 @@ class A {
 
     expect(named.linkedParams, matchesCompressed(r'''
         <span class="parameter" id="named-param-f">
-          \{
           <span class="type-annotation">
             <a href=".*/dart-core/int-class\.html">int</a>\?
           </span>
           <span class="parameter-name">f</span>
-          \}
         </span>
+        \}
```

Comment:
Where is the opening bracket for all the closing brackets in these tests?

Reply:
Good catch; these all moved to where you'd expect, based on where the closing ones moved. Updated the tests.